values.yaml

```
@@ -72,7 +72,20 @@ taiga:
   # emailSmtpPort: 587
   # emailSmtpUser: "user@example.com"
   # emailSmtpPassword: ""
-
+  ldap:
+    Enabled: False
+    use_tls: False
+    ldap_server: ldap://<ldap_server_host_name>
+    port: 389
+    bind_dn: "cn=something,ou=admins,ou=theouhere,ou=auth,o=ethz,c=ch"
+    base_dn: "basednhere"
+    password: "changeme"
+    username_attribute: "uid"
+    email_attribute: "mail"
+    full_name_attribute: "cn"
```

Also here would be nice to default to the image and plugin default
```suggestion
    full_name_attribute: "displayName"
```